eZ/Publish/Core/REST/Server/Exceptions/ContentFieldValidationException.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * File containing the ContentFieldValidationException tests.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
```

Comment:
Some time ago @andrerom mentioned that we should skip the line with `//autogentag//` as it is not used anymore.